src/en/wizden-staff/maintainer/adding-new-maintainers.md

```diff
@@ -0,0 +1,42 @@
+# Adding New Maintainers
+
+This document defines the process for adding new Maintainers to Space Station 14.
+
+## Requirements
+
+Maintainers represent Space Wizards as staff. Certain requirements must be met before a person can become a Maintainer:
+- A vote must be held according to the Voting Process outlined below.
+- Be a member of the Space Station 14 Discord server.
+- If the nominee has caused Administration issues in the past, these issues must be raised and discussed during the voting process. Administration issues include, but are not limited to, being banned from the game, banned from the Discord server, or having a history of being rude to other members of the community.
+- Have a good understanding of the Space Station 14 codebase and how it works. This must be raised and discussed during the voting process.
+- Have a solid understanding of the culture and values of the game and community
+
+### Voting Process
+
+To add a new Maintainer, a current Maintainer has to nominate the candidate. Once a person has been nominated, the following process should be followed:
+1. A vote will be created in the "Internal" category on the Space Station 14 Discourse, using the appropriate template. This vote will be open for one week, and all Maintainers will be notified of the vote.
+2. During this time, Maintainers should discuss the nomination, raise concerns, and ask any questions they may have. If there is a clear majority of Maintainers in favor or against the nomination, the vote may be closed early.
```

Comment:
The only mild concern I have left is that since the "close the vote early" part is left rather vague on the specifics, someone might feel aggrieved if the vote process was called before they could raise some concern they had, and then they could feel wronged. Probably not likely to ever come up, but, uh, if it ever does we will sure say "this should have been more specific"? What do you think

Reply:
Clear Majority in this case would be something like "It's been a day and 10 Maintainers said yes and no one said no"

Reply:
If this is meant to be for unanimous (preliminary) results, should it not say that?

Reply:
It should, I'll amend it.